### PR TITLE
 kube-proxy is part of the Kubernetes node components

### PR DESCRIPTION
**Pull request by gengwg**
kube-proxy is part of the Kubernetes node components not control plane. I think it's a typo.
ref: https://kubernetes.io/docs/concepts/overview/components/#node-components

### CODE REVIEW

**Comment by tengqm** [comment]
I don't think this is a typo or an error. You can for sure call kubelet, kube-proxy part of the control-plane in some contexts. These components are not centralized services, but they are integral parts of the control-plane that supports containerized apps. In a narrow sense, you can exclude node agents from control-plane, that is fine too.

**Comment by divya-mohan0209** [comment]
I tend to agree with @tengqm here but I think maybe rewording it to actually state what Qiming said would help clarify this for new users. WDYT @tengqm @gengwg ?

**Comment by gengwg** [comment]
My understanding about the concepts control plane and worker nodes may be incorrect. I was thinking worker nodes contains kubelet and kube-proxy, all others belong to control plane.

I think what Qiming means may be kubelet and kube-proxy works together with control plane components to achieve a common mission, which is to support containerized apps? In that sense, they are integral parts of the control-plane.

What Divya says also makes sense; it is a little confusing to readers (or maybe just me). The first sentence on the same page is: "In a Kubernetes cluster, **the components on the worker nodes - kubelet and kube-proxy** - need to communicate with Kubernetes control plane components, specifically kube-apiserver." However, late in the page it says: "Notable is **kube-proxy, which is part of the Kubernetes control plane** and runs on every node".  Maybe changing it to "Notable is **kube-proxy, which is another component on the worker nodes** and runs on every node" (or something similar) is more matching to the beginning sentence? *(not totally sure. a little confused currently :) i may need read more about k8s.)*

**Comment by tengqm** [comment]
I'm all for making the docs consistent. Thanks @gengwg .

File: content/en/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md [hunk 1]
```diff
@@ -448,7 +448,7 @@ A deployment-specific approval process for kubelet serving certificates should t
 
 All of TLS bootstrapping described in this document relates to the kubelet. However,
 other components may need to communicate directly with kube-apiserver. Notable is kube-proxy, which
-is part of the Kubernetes control plane and runs on every node, but may also include other components such as monitoring or networking.
+is part of the Kubernetes node components and runs on every node, but may also include other components such as monitoring or networking.
```
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md:451]: Aside: not all clusters use `kube-proxy` (some network plugins provide and use an alternative).

**Review by sftim — COMMENTED** [review]
/approve
